docs/source/key_components/dashboard_visualization.rst

```diff
@@ -0,0 +1,109 @@
+Dashboard Visualization
```

Comment:
since it's a new file, you need to add the file path to ``docs/source/index.rst``